packages/example-app-base/src/components/FeedbackList/FeedbackList.tsx

```diff
@@ -46,22 +46,22 @@ export class FeedbackList extends React.Component<IFeedbackListProps, IFeedbackL
   }
 
   public async getIssues(url: string): Promise<IListItem[]> {
-    let issueList: IListItem[] = [];
-
     const response = await fetch(url);
```

Comment:
this entire routine is a little buggy in that it does not handle pagination... (it simply assumes no pagination takes place)

Reply:
At least it doesn't throw a runtime exception AND not handle pagination tho ;)